commands/deploy.go

```diff
@@ -254,7 +261,7 @@ func addFileToTarWriter(filePath string, tarWriter *tar.Writer, prefix string) e
 		return fmt.Errorf(fmt.Sprintf("Could not write header for file '%s', got error '%s'", baseFilePath, err.Error()))
 	}
 
-	if !stat.IsDir() {
+	if !stat.IsDir() && stat.Mode()&os.ModeSymlink == 0 {
```

Comment:
Again, per multi-bit comment above.
```suggestion
	if !stat.IsDir() && stat.Mode()&os.ModeSymlink != os.ModeSymlink {
```